frontend/src/components/LabelForm/index.jsx

```diff
@@ -234,7 +234,7 @@ class LabelForm extends React.Component {
                         (Optional) Reason for Label:
                         <FormControl
                             componentClass="textarea"
-                            placeholder="Type reason here"
+                            placeholder={(this.state.label_reason === "") ? "Type reason here" : this.state.label_reason}
```

Comment:
So that makes sense for the history table, but it gets really annoying when you're trying to input a new reason in the annotation page. If you erase it it just pops back in.

Reply:
Yea I was commenting with respect to the history component.  If its too much of a hassle to differentiate which parent component the label form is in then I'm fine leaving as is.